### PR TITLE
SAK-43035 time => Instant, cover, util.FormattedText => api.Formatted…

### PR DESCRIPTION
**Pull request by axxter99**
…Text

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4846155&bid=16085096)

File: portal/portal-util/util/src/java/org/sakaiproject/portal/util/ErrorReporter.java [hunk 1]
```diff
@@ -362,7 +362,8 @@ protected void logAndMail(String bugId, String usageSessionId, String userId, St
 					+ pathInfo + rb.getString("bugreport.time") + ": " + time + "\n\n\n"
 					+ userComment + problemDisplay + placementDisplay + "\n\n" + requestDisplay;
 
-			EmailService.send(from, emailAddr, subject, body, emailAddr, null, null);
+			EmailService emailService =ComponentManager.get(EmailService.class);
```
[the remark this answers is not in the document]
Comment (axxter99) [reply portal/portal-util/util/src/java/org/sakaiproject/portal/util/ErrorReporter.java]: ouch )

File: portal/portal-util/util/src/java/org/sakaiproject/portal/util/ToolUtils.java [hunk 2]
```diff
@@ -27,28 +27,24 @@
...
 import org.sakaiproject.portal.api.Portal;
-import org.sakaiproject.util.Web;
-
+import org.sakaiproject.portal.api.PortalService;
 import org.sakaiproject.site.api.Site;
 import org.sakaiproject.site.api.SitePage;
 import org.sakaiproject.site.api.ToolConfiguration;
-import org.sakaiproject.tool.api.ActiveTool;
-import org.sakaiproject.tool.api.Tool;
+// Yes - crazy and brittle - but better in one place
```
[the remark this answers is not in the document]
Comment (axxter99) [reply portal/portal-util/util/src/java/org/sakaiproject/portal/util/ToolUtils.java]: patch -R

File: portal/portal-util/util/src/java/org/sakaiproject/portal/util/ErrorReporter.java [hunk 3]
```diff
@@ -362,7 +362,8 @@ protected void logAndMail(String bugId, String usageSessionId, String userId, St
...
+			emailService.send(from, emailAddr, subject, body, emailAddr, null, null);
```
[the remark this answers is not in the document]
Comment (axxter99) [reply portal/portal-util/util/src/java/org/sakaiproject/portal/util/ErrorReporter.java]: OK

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4846155&bid=16085096)

**Pull request merged**